packages/ui/package.json

```diff
@@ -88,6 +88,7 @@
     "eslint-plugin-react-refresh": "^0.4.3",
     "jest": "^29.4.2",
     "jest-environment-jsdom": "^29.4.2",
+    "lodash": "^4.17.21",
```

Comment:
Adding `lodash` as a `devDependency` is intended rather than a regular `dependency`?

I think it would be better just to bring what we need from `lodash`, in this case `cloneDeep`.

`"lodash.cloneDeep"` and also the types which should be `"@types/lodash.cloneDeep"`

Reply:
It seems there's no separate package for `lodash.cloneDeep`? this is 404 not found
https://registry.yarnpkg.com/lodash.cloneDeep

The reason being in dev dep is it's used only in tests.

Reply:
Maybe the version is different but we have it in kaoto-ui 😃 

https://github.com/KaotoIO/kaoto-ui/blob/main/packages/kaoto-ui/package.json#L61

Reply:
ahhhh it's `clonedeep` with lower `d` :rofl:

Reply:
OK done